### PR TITLE
Numerous changes from New() to Initialize()

### PR DESCRIPTION
**Pull request by tortellinitony**
All in the hippiestation folder. I did some other optimizations, such as list helpers.

### CODE REVIEW

File: hippiestation/code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -38,153 +36,16 @@
...
-			new /obj/item/weapon/throwing_star(src)
-			new /obj/item/weapon/throwing_star(src)
-			new /obj/item/weapon/throwing_star(src)
-			new /obj/item/weapon/throwing_star(src)
-			new /obj/item/weapon/throwing_star(src)
-			new /obj/item/weapon/storage/belt/chameleon(src) // Unique but worth at least 2 tc
-			new /obj/item/weapon/card/id/syndicate(src) // 2 tc
-			new /obj/item/device/chameleon(src) // 7 tc
-
-
```
Comment (ghost) [inline hippiestation/code/game/objects/items/weapons/storage/uplink_kits.dm:182]: Am I missing something here? Where did the random supply kits go?

Comment (tortellinitony) [reply hippiestation/code/game/objects/items/weapons/storage/uplink_kits.dm:182]: They are still in the main file. This was put here as a mistake.

**Review by ghost — APPROVED** [review]

File: hippiestation.dme [hunk 2]
```diff
@@ -2102,8 +2102,6 @@
 #include "hippiestation\code\game\objects\items\weapons\implants\implant_misc.dm"
 #include "hippiestation\code\game\objects\items\weapons\implants\implantcase.dm"
 #include "hippiestation\code\game\objects\items\weapons\implants\implanter.dm"
-#include "hippiestation\code\game\objects\items\weapons\storage\backpack.dm"
```
Comment (ghost) [inline hippiestation.dme:2105]: ?

Comment (tortellinitony) [reply hippiestation.dme:2105]: another thing that shouldn't have been done and will cause problems

Comment (ghost) [reply hippiestation.dme:2105]: Tony, when you say that these things shouldn't have been done, do you mean the removing part or the existing in the first place part?

**Pull request merged**